public/javascripts/SVLabel/css/svl-modal.css

```diff
@@ -74,7 +74,7 @@
 }
 
 .modal-foreground p {
-    font-size: 12pt;
```

Comment:
Please put this back to 12pt. I don't want other `<p>` text under .modal-foreground to be affected.